### PR TITLE
all: added README & make button inline

### PR DESCRIPTION
**Pull request by liharsw**
Make button inline and add new readme

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,28 +9,16 @@
...
-            <br> <br>
-            <button onclick="printHandover()">
-                Handover Form
-            </button>
-            <br> <br>
-            <button onclick="printTest()">
-                Test Printer
-            </button>
+            <button onclick="printCsd()">CSD Form</button>
+            <button onclick="printInvoice()">Invoice Form</button>
```
Comment (Rusydy) [inline index.html:19]: Give space between tag

File: README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# QZ Print Using JS
```
Comment (Rusydy) [inline README.md:1]: Please create simple description on the file

File: index.html (hunk 1 above)
Comment (liharsw) [reply index.html:19]: will update the tag

File: README.md (hunk 2 above)
Comment (liharsw) [reply README.md:1]: what description we should be used?

**Review by Rusydy — CHANGES_REQUESTED** [review]
Please add a description about what this PR all about

**Comment by Rusydy** [comment]
Please add new ignore file

**Pull request merged**